### PR TITLE
GH-1344: Prioritizing classes

### PR DESCRIPTION
**Pull request by klasocki**
Add possibility for including class weights in cross-entropy loss (when not using CRF) and using F-Beta as the evaluation metric

### CODE REVIEW

**Comment by djstrong** [comment]
Weights can be added also to `TextClassifier`.

**Comment by djstrong** [comment]
Class weighting is important in macro metrics.

**Comment by alanakbik** [comment]
@klasocki thanks for adding this and please excuse the delay in checking this PR! I've ran some local tests with different parameters and it works well for prioritizing classes and controlling the precision/recall trade-off so lot's of people will surely find this useful!

**Comment by alanakbik** [comment]
:+1:

**Comment by klasocki** [comment]
> @klasocki thanks for adding this and please excuse the delay in checking this PR! I've ran some local tests with different parameters and it works well for prioritizing classes and controlling the precision/recall trade-off so lot's of people will surely find this useful!

No problem whatsoever! Thank you for caring about the quality of Flair. And I'm very glad you found it useful 😄

**Comment by BasLaanPAM** [comment]
Thank you for adding this feature, it is very helpful. I was wondering if there is a specific reason it was only added in cross-entropy loss and not the viterbi loss (using CRF)? Having a quick look at the [code ](https://github.com/flairNLP/flair/blob/8d27a383810455cb45d650e7fe2003384780ae84/flair/models/sequence_tagger_utils/viterbi.py#L79) we should be able to multiple this loss function by weights as well.
